### PR TITLE
WT-9045 Change minutes to uint64 to avoid overflow in __wt_gen_drain

### PR DESCRIPTION
**Pull request by clarissecheah**

### CODE REVIEW

File: src/support/generation.c [hunk 1]
```diff
@@ -169,7 +168,7 @@ __wt_gen_drain(WT_SESSION_IMPL *session, int which, uint64_t generation)
 #define WT_GEN_DRAIN_TIMEOUT_MIN 4
                 if (time_diff_ms > minutes * WT_MINUTE * WT_THOUSAND) {
                     __wt_verbose_notice(session, WT_VERB_GENERATION,
-                      "%s generation drain waited %u minutes", __gen_name(which), minutes);
+                      "%s generation drain waited %" PRIu64 "minutes", __gen_name(which), minutes);
```
Comment (korteland) [inline src/support/generation.c:171]: this is missing a space before "minutes" now

Comment (clarissecheah) [reply src/support/generation.c]: Fixed that!

**Review by korteland — APPROVED** [review]
Thanks, this looks good now!

File: src/support/generation.c [hunk 2]
```diff
@@ -107,9 +107,8 @@ __wt_gen_drain(WT_SESSION_IMPL *session, int which, uint64_t generation)
     struct timespec start, stop;
     WT_CONNECTION_IMPL *conn;
     WT_SESSION_IMPL *s;
-    uint64_t time_diff_ms, v;
+    uint64_t time_diff_ms, v, minutes;
```
Comment (agorrod) [inline src/support/generation.c:110]: We generally alphabeticalize anything we can - which makes `minutes` the first thing in the list of local variables here.

**Review by agorrod — CHANGES_REQUESTED** [review]
Thanks - I've got one minor request, otherwise this looks good.

File: src/support/generation.c (hunk 2 above)
Comment (clarissecheah) [reply src/support/generation.c:110]: Oh yes, sure thing. Thanks for pointing that out Alex!

**Review by agorrod — APPROVED** [review]
Lgtm - thanks!

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Pull request merged**